10300.xml

```diff
@@ -0,0 +1,87 @@
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>LWM2M Meta Object</Name>
+    <Description1><![CDATA[Use this meta object to provide one or more locations for data model XML definitions for vendor-specific devices. ]]></Description1>
+    <ObjectID>10300</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10300</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+      <Item ID="0">
+        <Name>ObjectID</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units></Units>
+        <Description><![CDATA[Object ID of the vendor-specific object]]></Description>
+      </Item>
+      <Item ID="1">
+        <Name>ObjectURN</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units></Units>
+        <Description><![CDATA[Object URN of the vendor-specific object]]></Description>
+      </Item>
+      <Item ID="4000">
+        <Name>ObjectInstanceHandle</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Objlnk</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units></Units>
+        <Description><![CDATA[Reusable LWM2M resource. This can be used to differentiate object instances having the same object ID and URN (for example, for Object IDs in the range 26241 - 32768)]]></Description>
+      </Item>
+      <Item ID="2">
+        <Name>URL</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Multiple</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units></Units>
+        <Description><![CDATA[Location of the XML file containing the object definition of the vendor-specfic object. Multiple instances can be used if several alternative locations are supplied.]]></Description>

```

Comment:
That would be good to define. I'd also recommend that in addition to the ObjectInstanceHandle that LWM2M already defines as a reusable resource, and the ObjectURN that you are proposing to turn into a reusable resource, the ObjectID can also be defined as a reusable resource.

Reply:
This was discussed in the IPSO call today and a third option was suggested: we could potentially make a new data type for URLs that could be used instead of String. Would that be good here? However, that could come earliest with next version of the LwM2M spec.